src/options.js

```diff
@@ -0,0 +1,34 @@
+const defaultOptions = {
+  drawing: true,
+  interactive: false,
+  position: 'top-left',
+  keybindings: true,
+  displayControlsDefault: true,
+  styles: {},
+  controls: {}
+}
+
+const showControls = {
+  point: true,
+  line: true,
+  polygon: true,
+  trash: true
+};
+
+const hideControls = {
+  point: false,
+  line: false,
+  polygon: false,
+  trash: false
+};
+
+module.exports = function(options = {controls: {}}) {
+
+  if (options.displayControlsDefault === false) {
```

Comment:
Yea, I'd much rather have it be `=== false`. It is only false if false is sent, 0, undefined and null don't set it.